scarlet/src/test/java/com/tinder/scarlet/ScarletTest.kt

```diff
@@ -73,6 +74,64 @@ internal class ScarletTest {
         then(service).should().execute(ExampleService::class.java.getDeclaredMethod("receive"), emptyArray())
     }
 
+    @Test
+    fun create_hashCode_shouldEqualServiceInstanceHashCode() {
+        // Given
+        val service = mock<Service>()
+        given(serviceFactory.create(ExampleService::class.java)).willReturn(service)
+        val exampleService = scarlet.create<ExampleService>()
+
+        // When
+        val hashCode = exampleService.hashCode()
+
+        // Then
+        assertThat(hashCode).isEqualTo(service.hashCode())
+    }
+
+    @Test
+    fun create_toString_shouldProduceCorrectValue() {
+        // Given
+        val service = mock<Service>()
+        given(serviceFactory.create(ExampleService::class.java)).willReturn(service)
+        val exampleService = scarlet.create<ExampleService>()
+
+        // When
+        val toString = exampleService.toString()
+
+        // Then
+        assertThat(toString)
+            .isEqualTo("Scarlet service implementation for com.tinder.scarlet.ScarletTest\$Companion\$ExampleService")
+    }
+
+    @Test
+    fun create_equals_shouldEqualSelf() {
+        // Given
+        val service = mock<Service>()
+        given(serviceFactory.create(ExampleService::class.java)).willReturn(service)
+        val exampleService = scarlet.create<ExampleService>()
+
+        // When
+        val equalsSelf = exampleService.equals(exampleService)
+
+        // Then
+        assert(equalsSelf) { "equals must be reflexive" }
```

Comment:
Could you use assertJ here?

i.e.
`assertThat(equalsSelf).describedAs("equals must be reflexive").isTrue().`